platformio.ini

```diff
@@ -35,5 +35,12 @@ build_flags =
 	-DBOARD_HAS_PSRAM
 	-UARDUINO_USB_CDC_ON_BOOT
 	-mfix-esp32-psram-cache-issue
-	-DCORE_DEBUG_LEVEL=5
-;	-DLAST_BUILD_TIME=$UNIX_TIME
+
+[env:sa868_vhf_open_edition]
+build_flags= ${env.build_flags} -DSA868_OPEN_EDITION
+
+[env:sa868_vhf_open_edition_debug]
```

Comment:
@lyusupov maybe its better to do something like this?
separate build for each module?